pkg/operation/botanist/kubeapiserver.go

```diff
@@ -156,7 +156,6 @@ func (b *Botanist) DefaultKubeAPIServer(ctx context.Context) (kubeapiserver.Inte
 			StaticTokenKubeconfigEnabled:   b.Shoot.GetInfo().Spec.Kubernetes.EnableStaticTokenKubeconfig,
 			Version:                        b.Shoot.KubernetesVersion,
 			VPN: kubeapiserver.VPNConfig{
-				ReversedVPNEnabled:                   b.Shoot.ReversedVPNEnabled,
```

Comment:
I assume you can drop L420-L423 from this file. Then also delete https://github.com/gardener/gardener/blob/master/charts/images.yaml#L92-L95. Also drop  the `VPNSeed` field in the `kubeapiserver.Images` struct.